cellxgene_resources/cellxgene_mods.py

```diff
@@ -951,33 +952,30 @@ def evaluate_var_df(adata):
         report('Features in var.index are gene symbols and/or contain deprecated Ensembl IDs', 'ERROR')
         return
     valid = True
-    obs_organisms = adata.obs['organism_ontology_term_id'].unique().tolist()
+    uns_organism = adata.uns['organism_ontology_term_id']
     var_organisms = [o.value for o in var_organism_objs]
 
     if 'NCBITaxon:2697049' in var_organisms:
         report('There are covid genes present in var')
         var_organisms.remove('NCBITaxon:2697049')
-    if 'NCBITaxon:2697049' in obs_organisms:
-        report('Covid is found in obs metadata', 'ERROR')
-        obs_organisms.remove('NCBITaxon:2697049')
+    if 'NCBITaxon:2697049' == uns_organism:
+        report('Covid is found in uns metadata', 'ERROR')
```

Comment:
A different message could be more direct to help diagnose the error. Maybe something like "Covid is not a supported uns.organism"